tools/misc/browse_dataset.py

```diff
@@ -79,6 +79,8 @@ def build_data_cfg(config_path, skip_type, aug, cfg_options):
         for i in range(len(cfg.train_pipeline)):
             if cfg.train_pipeline[i]['type'] == 'LoadAnnotations3D':
                 show_pipeline.insert(i, cfg.train_pipeline[i])
+        # Collect points as well as labels
+        show_pipeline[-1] = cfg.train_pipeline[-1]
```

Comment:
Maybe need an if to make sure the last pipeline is Collect OR can we directly modify the eval_pipeline?
The current modification is a little hack?

Reply:
It's true that the last pipeline is Collect3D is the last pipeline for all configs,  but I suggest that it's better to be written like the way of updating 'LoadAnnotations3D'.